frontend/src/Components/CustomizedInputBase.js

```diff
@@ -29,14 +30,14 @@ export function CustomizedInputBase(props) {
     // console.log(input)
     props.apiCalled(true)
 
-    // const res = await axios.post('http://127.0.0.1:5000/api/v1/dashboard', {
-    //   input: input,
-    //   subgraph: subgraph,
-    // })
+    const res = await axios.post('http://127.0.0.1:5000/api/v1/dashboard', {
```

Comment:
LGTM. Can we have this read from .env also?

Reply:
YEP, one sec

Reply:
DONE